hack/lib.sh

```diff
@@ -163,8 +162,7 @@ function storage_admin_set_up() {
     --member=serviceAccount:"${pubsub_service_account}"@"${project_id}".iam.gserviceaccount.com \
     --role roles/storage.admin
 
-  curl -s -X GET -H "Authorization: Bearer \`GOOGLE_APPLICATION_CREDENTIALS=${pubsub_service_account_key_temp} gcloud auth application-default print-access-token\`" \
-    "https://www.googleapis.com/storage/v1/projects/${project_id}/serviceAccount"
+  # Get service account credentials from: https://cloud.google.com/storage/docs/json_api/v1/projects/serviceAccount/get/
```

Comment:
I would phrase this something like:

```suggestion
  # We assume the service account's name is in the form '<project-number>@gs-project-accounts.iam.gserviceaccount.com',
  # because it has been for all projects we've encountered. However, nothing requires this
  # format. For the actual email, execute this API request:
  # https://cloud.google.com/storage/docs/json_api/v1/projects/serviceAccount/get/
```